src/color/tier.json

```diff
@@ -81,6 +81,22 @@
               "wcag": "n/a",
               "deprecated": false
             }
+          },
+          "lounge": {
+            "value": "{color.brand.midnight.400.value}",
+            "public": true,
+            "onDark": true,
+            "usage": "Longe tier color for Alaska Airlines",
+            "wcag": "n/a",
+            "deprecated": false
+          },
+          "loungeplus": {
+            "value": "{color.brand.loungeplus.value}",
+            "public": true,
+            "onDark": true,
```

Comment:
```suggestion
```

Reply:
I've resolved your feedback. Please revalidate the changes. Thank you